examples/stable/test/property/ic_api/json/test/test.ts

```diff
@@ -0,0 +1,3 @@
+import { getTests } from './tests';
+
+describe('JSON property tests', getTests());
```

Comment:
This is very incorrect, you're supposed to call runTests. Look at the many examples in the ic_api directory and do what they do.